pytest_motor/plugin.py

```diff
@@ -33,21 +34,21 @@ async def root_directory(pytestconfig: PytestConfig) -> Path:
 
 
 @pytest.fixture(scope='session')
-async def mongod_binary(root_directory: Path) -> Path:  # pylint: disable=redefined-outer-name
+def mongod_binary(root_directory: Path) -> Path:  # pylint: disable=redefined-outer-name
     """Return a path to a mongod binary."""
     destination: Path = root_directory.joinpath('.mongod')
     mongod_binary_filename = destination.joinpath(
-        'mongodb-linux-x86_64-ubuntu1804-4.4.6/bin/mongod')
+        f'{__mongo_exec()}/bin/mongod')
 
     if not mongod_binary_filename.exists():
-        download_url = 'https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.4.6.tgz'
+        download_url = f'https://fastdl.mongodb.org/osx/{__mongo_exec()}.tgz'
```

Comment:
Is `/linux` needed for this to still work on Ubuntu?

Reply:
Yes, that's wrong it varies depending on the platform. I'm fixing it.